### DISCUSSION TITLE
NumTaps query

### DISCUSSION THREAD

**Discussion by limzw** [Mapsui/Mapsui#2559] [Q&A]
Hello, i am trying to get the number of times i click on a PointFeature, and i was trying to make use of the NumTaps property. I am using the following code
```
if(e.MapInfo?.Feature != null)
{
    Debug.WriteLine(e.NumTaps);
}
```
When i click only once, the NumTaps value printed is 1, which is correct. However, when i double click quickly (with my mouse) on my PointFeature, i noticed that 3 values are printed, 1, 2 and then 1 again.  Is this supposed to be the case?

**Comment by pauldendulk** [comment]
It is not supposed to work like that but this is something that won't be fixed in v4. There has been a big rewrite of everything related to mouse/touch manipulation in v5. A first beta will be released the coming weeks.

A workaround could be to use the tap-events or gestures of the platform and request MapControl.GetMapInfo (which I assume you need) in your handler.

What platform are you using?

**Reply by limzw** [reply]
Ok thank you for clarifying, I am using Avalonia!
